### PR TITLE
updates to core theme doc

### PR DESCRIPTION
**Pull request by rdlauer**
Includes numerous tweaks to code samples to make them more
copy-and-paste friendly. Includes iOS and Android screenshots for most
code examples as well.

### CODE REVIEW

File: docs/ui/theme.md [hunk 1]
```diff
@@ -32,19 +31,20 @@ The NativeScript project provides a core theme that you can add to any of your p
...
     - [TabViews](#tabviews)
+* [SASS Usage](#sass-usage)
 * [Status Bar Considerations](#status-bar-considerations)
 * [Uninstalling](#uninstalling)
 * [Contributing](#contributing)
 
 ## Installation
 
-As of NativeScript version 2.4, the core theme is installed by default when you start an app using `tns create`. However, if you wish to add the theme to a NativeScript app built before 2.4, run the following command:
+As of NativeScript version 2.4, the core theme is installed by default when you start an app using `tns create`. However, if you wish to add the theme to a NativeScript app built **before** 2.4, run the following command:
```
Comment (tjvantoll) [inline docs/ui/theme.md]: We can probably drop the 2.4 bit at this point. No one is coming to this page with a pre-2.4 app.

File: docs/ui/theme.md [hunk 2]
```diff
@@ -32,19 +31,20 @@ The NativeScript project provides a core theme that you can add to any of your p
...
 
 ```
 npm install nativescript-theme-core --save
 ```
 
-After you install the theme, you need to add a single `@import` CSS rule to your `app.css` file in order to include the theme in your app. But first you need to decide which of the theme’s color schemes you’d like to use.
+After you install the theme on a pre-2.4 app, you need to add a single `@import` CSS rule to your `app.css` file in order to include the theme in your app. But first you need to decide which of the theme’s color schemes you’d like to use.
```
Comment (tjvantoll) [inline docs/ui/theme.md]: same

File: docs/ui/theme.md [hunk 3]
```diff
@@ -690,28 +653,141 @@ The NativeScript core theme includes a class name for styling Slider UI componen
...
+```
+.
+├── custom.scss
+├── custom.css
+└── app.component.ts
+```
+
+app.component.ts
+
+```
```
Comment (tjvantoll) [inline docs/ui/theme.md]: You didn’t change this, but can you add TypeScript highlighting? Aka add the word TypeScript after the three backticks.

File: docs/ui/theme.md [hunk 4]
```diff
@@ -690,28 +653,141 @@ The NativeScript core theme includes a class name for styling Slider UI componen
...
+.
+├── _app-common.scss
+├── app.android.scss
+└── app.ios.scss
+```
+
+After that, paste the following code into your `app.android.scss` file.
+
+``` SCSS
```
Comment (tjvantoll) [inline docs/ui/theme.md]: Can you change the few SCSS highlighting to just CSS? The docs won’t highlight blocks marked as SCSS.

**Review by tjvantoll — APPROVED** [review]
Thanks @rdlauer. This was a lot of monotonous work and is much appreciated 😄

**Comment by tjvantoll** [comment]
👍🏻 from me

**Comment by tjvantoll** [comment]
Merged with https://github.com/NativeScript/docs/commit/0237d7199c56e6d7abb1ac44275f39c346fd4127 and https://github.com/NativeScript/docs/commit/ef1f9a85ce0ad22a25b043c050cc70f8dfb3e359.

**Comment by pedromorgan** [comment]
looks great here.. ta ;-)

**Pull request merged**